### ISSUE TITLE
Engine v0.16.0, Issue Limit per Rule: not enforced? Or, wrong documentation?

### ISSUE THREAD

**Issue by mhungria** [cdisc-org/cdisc-rules-engine#1718]
### Standard

CDISC SDTM/SDTMIG

### Reference Rule ID(s)

FB2604

### Conformance Rule ID(s) (if published in CORE)

across all

### JIRA Ticket

https://jira.cdisc.org/projects/CORERULES/issues/CORERULES-

### CLI Command Used (if applicable)

python core.py -s SDTM -v 3.4 -d /path/to/datasets

### Rule YAML

```YAML
NA
```

### Attach any output report and/or log file(s)

<img width="902" height="455" alt="Image" src="https://github.com/user-attachments/assets/d63f38ea-a338-4bd4-b9b4-590fd4a31c1b" />

<img width="894" height="301" alt="Image" src="https://github.com/user-attachments/assets/ef74afd9-2703-4433-bc7c-1c79f9e81070" />

<img width="962" height="306" alt="Image" src="https://github.com/user-attachments/assets/17316064-9f88-4de6-9d01-089f1f56f3df" />

### Expected output

At most 10 rows per rule as Issue Details, according to the limit set and indicated in the Conformance Details.

Note: I think the total # of issues should be left as is in the Issue Summary.
